.travis.yml

```diff
@@ -16,10 +16,9 @@ services:
   - postgresql
 
 before_script:
-  - cp config/database.travis.yml config/database.yml
-  - psql -c "CREATE USER rails-bootstrap WITH PASSWORD 'rails-bootstrap';" -U postgres
-  - psql -c "DROP DATABASE IF EXISTS rails-bootstrap_test;" -U postgres
-  - psql -c "CREATE DATABASE rails-bootstrap_test ENCODING 'utf8';" -U postgres
+  - psql -c "CREATE USER federicolopezcanas WITH PASSWORD 'password';" -U postgres
```

Comment:
Its not bad that you used your user role. But if its a project for a client or one where there are many developers you can use the database.travis.yml for generic setup